RELEASING.md

```diff
@@ -110,5 +110,5 @@ Foxy | Edifice | 0.223.X
 1. Use [release-tools](https://bitbucket.org/osrf/release-tools)'s script to launch jenkins jobs:
     ```
     cd release-tools/bloom
-    ./ros_ign_bridge-release.py.bash 0.8.0 https://github.com/osrf/ros_ign-release <ros_distro> <token> --ignition-version <version_name>
+    ./ros_gz_bridge-release.py.bash 0.8.0 https://github.com/gazebosim/ros_gz-release <ros_distro> <token> --gz-version <version_name>
```

Comment:
We have to migrate this file in `release-tools`

And it looks like this repo hasn't moved, so we can keep the old URL for now. I actually don't know if this is still expected to work.

https://github.com/osrf/ros_ign-release

Reply:
The repo change has been reverted, but the file has been migrated